pandas/tests/indexes/datetimes/test_constructors.py

```diff
@@ -1042,10 +1042,18 @@ def test_datetimeindex_constructor_misc(self):
         arr = np.array(["1/1/2005", "1/2/2005", "1/3/2005", "2005-01-04"], dtype="O")
         idx4 = DatetimeIndex(arr)
 
-        arr = to_datetime(["1/1/2005", "1/2/2005", "1/3/2005", "2005-01-04"])
+        # Can't be parsed consistently, need to parse each element individually
```

Comment:
I think better to just make these explicit with the new values or even remove if there is no value